source/documentation/monitoring_apps/metrics.md

```diff
@@ -123,3 +123,84 @@ If you are not receiving any metrics, check the [logs](/monitoring_apps.html#log
+
+1. Sign into the GOV.UK PaaS admin tool for [London](https://admin.london.cloud.service.gov.uk/) or [Ireland](https://admin.cloud.service.gov.uk/).
+1. Select the __Organisation__ and __Space__.
+1. Select the __Backing services__ tab and select your backing service.
+1. Select the __Metrics__ tab.
+
+### Metrics definitions - PostgreSQL and MySQL
+
+PostgreSQL and MySQL are relational database management systems.
+
+#### Free disk space
+
+How much hard disk space your database has remaining. Your database will stop working if it runs out of disk space. Upgrade your service plan to get more disk space.
+
+#### CPU utilisation
+
+How much computational work your database is doing. If you think your CPU utilisation is too high, you can optimise your database queries or upgrade your service plan.
+
+#### Open connections
+
+How many open connections there are to your database. Unexpectedly high values may indicate problems with your apps' connection management, or that you need to upgrade your service plan. Unexpectedly low values may indicate the database is unavailable, or that your apps cannot connect to the database.
+
+#### Available Memory
+
+How much memory the virtual machine your database is running on has remaining. Unexpectedly low values may indicate you need to optimise your database queries or upgrade your service plan.
+
+#### Read and write operations per second
+
+How many read and write operations per second (IOPS) your database is performing per second. Databases have a limit of 3 IOPS per gigabyte of database hard disk size. For example, a 100 gigabyte database has a 300 IOPS limit. If your database is close to its IOPS limit, you can upgrade your service plan.
+
+See the [documentation on Amazon RDS database instance storage](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#CHAP_Storage.IO.Credits) for more information.
+
+### Metrics definitions - Redis
+
+Redis is an open source in-memory data store.
+
+#### CPU utilisation
+
+How much computational work your Redis service instance is doing. If you think your CPU utilisation is too high, you can optimise your database queries or upgrade your service plan.
+
+#### Memory used
+
+How much memory your Redis service instance is using to run itself and to store your app data. Memory limit is pre-set based on service plan and you cannot change this memory limit. If your service instance reaches its memory limit, it will start [evicting data keys](monihttps://redis.io/topics/lru-cache). This may indicate that you need to review the size of the data you're storing, or that you need to upgrade your service plan.
+
+#### Swap memory used
+
+If your Redis service instance is running low on memory, it will start to swap memory onto the hard disk. To reduce memory swapping, you can reduce the number of apps using your service instance or upgrade your service plan.
```

Comment:
> To reduce memory swapping, you can reduce the number of apps using your service

It's more about making an effort to reduce the amount of memory being used than reducing the number of apps using the service.

Reply:
will change